templates/strings/flat-swift3.stencil

```diff
@@ -34,6 +34,8 @@ import Foundation
   {{accessModifier}} static func {{string.key|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}}({% call parametersBlock string.types %}) -> String {
     return {{enumName}}.tr("{{table}}", "{{string.key}}", {% call argumentsBlock string.types %})
   }
+  {% elif param.localizeFunction %}
```

Comment:
Worth a stencil comment `{#  custom localization function is mostly used for in-app lang selection,  so we want the loc to be recomputed at each call for those (hence  the computed car) #}`?